reformcloud-library/src/main/java/systems/reformcloud/meta/server/versions/SpigotVersions.java

```diff
@@ -155,8 +157,18 @@
     /**
      * GlowStone Versions
      */
```

Comment:
Adding GlowStone Versions needs a reformat of the Server startup because of the config files

Reply:
Everything is now Added.